packages/atomic/cypress/integration/common-assertions.ts

```diff
@@ -29,6 +30,29 @@ export function assertAccessibility(
       });
     }
   });
+
+  it('every interactive element with innerText and an aria label passes WCAG success criterion 2.5.3', () => {
```

Comment:
I made this test extremely permissive. It will only check letters. The reason is that it's pretty frequent for innerText to contain other artifacts like the count in parenthesis.